test/runtests.jl

```diff
@@ -2117,3 +2117,7 @@ finally
     rm(load_path, recursive=true)
     rm(load_cache_path, recursive=true)
 end
+
+using Aqua
+
+Aqua.test_all(Unitful, ambiguities=false, unbound_args=false)
```

Comment:
```suggestion
Aqua.test_all(Unitful, ambiguities=false, unbound_args=false, piracy=VERSION≥v"1.8")
```
On older Julia versions, the `zero(x::AbstractArray{Union{Missing, T}}) where T<:Unitful.AbstractQuantity` method is flagged for piracy. Therefore, I would suggest to only test for piracy on Julia ≥1.8.